src/IsoSprite.js

```diff
@@ -117,13 +117,13 @@ Phaser.Plugin.Isometric.IsoSprite.prototype.resetIsoBounds = function () {
     var asx = Math.abs(this.scale.x);
     var asy = Math.abs(this.scale.y);
 
-    this._isoBounds.widthX = Math.round(Math.abs(this.width) * 0.5) * asx;
-    this._isoBounds.widthY = Math.round(Math.abs(this.width) * 0.5) * asx;
+    this._isoBounds.widthX = Math.round(Math.abs(this.width) * 0.5 * Math.sqrt(5)/2) * asx;
+    this._isoBounds.widthY = Math.round(Math.abs(this.width) * 0.5 * Math.sqrt(5)/2) * asx;
     this._isoBounds.height = Math.round(Math.abs(this.height) - (Math.abs(this.width) * 0.5)) * asy;
 
-    this._isoBounds.x = this.isoX + (this._isoBounds.widthX * -this.anchor.x) + this._isoBounds.widthX * 0.5;
-    this._isoBounds.y = this.isoY + (this._isoBounds.widthY * this.anchor.x) - this._isoBounds.widthY * 0.5;
-    this._isoBounds.z = this.isoZ - (Math.abs(this.height) * (1 - this.anchor.y)) + (Math.abs(this.width * 0.5));
+    this._isoBounds.x = this.isoX + (this._isoBounds.widthX * -this.anchor.x) + (this._isoBounds.widthX * 0.5) - (this._isoBounds.widthX * this.anchor.y);
+    this._isoBounds.y = this.isoY + (this._isoBounds.widthY * this.anchor.x) - (this._isoBounds.widthY * 0.5) - (this._isoBounds.widthY * this.anchor.y);
```

Comment:
Since the fix suggested in issue #51 includes changing this.anchor.**x** to this.anchor.**y** in this line: are you sure it is intended here to keep this.anchor.**x**?